index.js

```diff
@@ -64,6 +64,7 @@ function fromUrl (giturl, opts) {
         var pathmatch = gitHostInfo.pathmatch
         var matched = parsed.path.match(pathmatch)
         if (!matched) return
+        /* istanbul ignore else */
```

Comment:
Could this be tested with a url where the host matches, but the path isn't a match?  Like `https://github.com/isaacs`?

Possibly I'm misreading this bit, and it is actually just a safety guard that never hits.  But in that case, it'd be preferable to make that explicit.

```js
/*istanbul ignore if */
if (matched[1] === null || matched[1] === undefined) {
  throw new Error('this should be impossible')
}
```